app/dashboard/src/components/Devtools/EnsoDevtoolsProvider.tsx

```diff
@@ -0,0 +1,73 @@
+/**
+ * @file
+ * This file provides a zustand store that contains the state of the Enso devtools.
+ */
+import type { PaywallFeatureName } from '#/hooks/billing'
+import * as zustand from 'zustand'
+
+/**
+ * Configuration for a paywall feature.
+ */
+export interface PaywallDevtoolsFeatureConfiguration {
+  readonly isForceEnabled: boolean | null
+}
+
+// =========================
+// === EnsoDevtoolsStore ===
+// =========================
+
+/** The state of this zustand store. */
+interface EnsoDevtoolsStore {
+  readonly showVersionChecker: boolean | null
```

Comment:
what is the difference between feature flags and the things that go in the devtools store?

Reply:
Feature flags can (at least in the future) be sent back from the server/by sending a special link. It's more like turning beta features on/off to collect feedback and the devtools are an internal feature only for us (not supposed to be visible to users). See devtools more like an interface (or admin) to quickly edit feature toggles

Reply:
ah, makes sense. we may want to add a settings page for it later (iirc there is one in the figma design called "feature preview")